package.json

```diff
@@ -26,5 +26,31 @@
   "bugs": {
     "url": "https://github.com/secondstreet/talker.js/issues"
   },
-  "homepage": "https://github.com/secondstreet/talker.js"
+  "homepage": "https://github.com/secondstreet/talker.js",
+  "lint-staged": {
+    "linters": {
+      "src/**/*.ts": [
+        "prettier --write",
+        "git add"
+      ],
+      "**/*.json": [
+        "prettier --write",
+        "git add"
+      ]
+    }
+  },
+  "devDependencies": {
+    "husky": "^1.2.0",
+    "lint-staged": "^8.1.0",
+    "prettier": "^1.15.2",
+    "ts-loader": "^5.3.1",
+    "tslint": "5.7.0",
+    "tslint-config-prettier": "1.9.0",
+    "typescript": "3.1.2",
+    "webpack": "^4.26.1",
+    "webpack-cli": "^3.1.2"
+  },
+  "dependencies": {
+    "es6-promise": "4.2.4"
```

Comment:
Just noticed that this and a few of the ones in devDeps don't have a `^` in front of the version; any particular reason? Would be nice to have them all defined the same way if not. (I noticed because the yarn lock update in ss-embed was listing both `^4.2.4` and `4.2.4` for `es6-promise` separately).

Reply:
For devDependencies, it mostly came down to the ones I don't trust to follow semver. TypeScript introduces breaking changes outside of major version releases, so I didn't allow it (or tslint+friends) to auto-upgrade.

For dependencies, it's because this is a public library and I wanted to pin very specific dependency versions as "actually tested to work".

Reply:
After talking further offsite, let's go with `~` -- it should be safe enough.